e2e-tests/js_scripts/0008-update-runtime-multisig-scheduler.js

```diff
@@ -0,0 +1,153 @@
+};
+const fs = require('fs');
+
+async function run(nodeName, networkInfo, args) {
+    const {wsUri, userDefinedTypes} = networkInfo.nodesByName[nodeName];
+    const api = await zombie.connect(wsUri, userDefinedTypes);
+
+    const ALICE = '5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY';
+    const BOB   = '5FHneW46xGXgs5mUiveU4sbTyGBzmstUspZC92UhjJM694ty';
+    const CHARLIE = '5FLSigC9HGRKVhB9FiEo4Y3koPsNmBmLJbpXg2mp1hXcS59Y';
+
+    // Use the keyring to generate accounts
+    const keyring = new zombie.Keyring({ type: 'sr25519' });
+    const alice = keyring.addFromUri('//Alice');
+    const bob = keyring.addFromUri('//Bob');
+    const charlie = keyring.addFromUri('//Charlie');
+    const ACME = 1_000_000_000_000_000_000n;
+
+    /*****************************************************************************************************
+     *************************************** CREATE MULTISIG ACCOUNT *************************************
+     *****************************************************************************************************/
+    const threshold = 2;
+    const multisigAddress = util.createKeyMulti([alice.address, bob.address, charlie.address], threshold);
+    const SS58Prefix = 42;
+    const Ss58MultiAddress = util.encodeAddress(multisigAddress, SS58Prefix);
+    console.log(`multisigAddress ${Ss58MultiAddress}`);
+
+
+    /*****************************************************************************************************
+     *************************************** SET MULTISIG AS SUDO ACCOUNT ********************************
+     *****************************************************************************************************/
+
+    const newSudoKey = api.tx.sudo.setKey(Ss58MultiAddress);
+    await submitExtrinsic(newSudoKey, alice, BlockUntil.Finalized, undefined);
+
+    const newSudoKeyOption = await api.query.sudo.key();
+    if (newSudoKeyOption.isSome && newSudoKeyOption.unwrap().toString() !== Ss58MultiAddress) {
+        return ReturnCode.ErrWrongSudoAccount;
+    }
+
+    /*****************************************************************************************************
+     *************************************** GET CURRENT RUNTIME VERSION******* **************************
+     *****************************************************************************************************/
+
+    const currentRuntimeVersion = (await api.rpc.state.getRuntimeVersion()).specVersion.toNumber();
+
+    /*****************************************************************************************************
+     *************************************** SCHEDULE RUNTIME UPGRADE*************************************
+     *****************************************************************************************************/
+
+    // Retrieve the runtime to upgrade
+    const sudoAccount = await api.query.sudo.key()
+    const code = fs.readFileSync('./new-runtime/nh_runtime.wasm').toString('hex');
```

Comment:
Which is the version of the test runtime ? Be sure to set it high enough such that this test always pass considering we are going to normally upgrade the runtime versions

Reply:
+1

I guess that generate a wasm with `spec_version = 4_000_000_000` should be something that'll work forever.